internal/config/config_test.go

```diff
@@ -153,14 +153,21 @@ func TestGetGVKs(t *testing.T) {
 				Kind:    "Pod",
 			}},
 		},
-		"inexistent-group": {
+		"inexistent-group-with-version-set": {
```

Comment:
spelling nit: nonexistent rather than inexisting (sorry, I know everybody hates spelling nits!)

Reply:
Although TIL that's a real word 🤯 

Makes this nit even less valuable. Feel free to ignore this completely.

Reply:
Interesting! I didn't know that nonexistent was preferred over inexistent! 😅 

Also I'm happy about spelling nits. Either it was a typo - definitely something that needs to be fixed 💯 - or I'm learning some new english 😁

Reply:
What I was saying poorly in my 2nd comment is that google says it's a real word, which I never knew before!